yesod-auth/Yesod/Auth/Message.hs

```diff
@@ -60,6 +60,7 @@ data AuthMessage =
     | ProvideIdentifier
     | SendPasswordResetEmail
     | PasswordResetPrompt
+    | CurrentPassword
```

Comment:
Hey, I was looking to send an email to get translations. Is this the mailing list you were referring to? https://mail.haskell.org/mailman/listinfo/web-devel